.github/workflows/rerun-failed-run.yml

```
@@ -12,10 +12,22 @@ on:
       repository:
         description: GitHub repository (ORG/REPO_NAME)
         required: true
+      notify-back-on-error:
+        description: \
```

not sure this is what you want for the multiline YAML:
```suggestion
        description: |-
```